ITunesSearch/ITunesSearch/SearchResult.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  SwiftResult.swift
+//  ITunesSearch
+//
+//  Created by Taylor Lyles on 5/14/19.
+//  Copyright © 2019 Taylor Lyles. All rights reserved.
+//
+
+import Foundation
+
+struct SearchResult: Codable {
+	var title: String
```

Comment:
The JSON returns some nil values for these at different calls. Make them optional and it tells the decoder to ignore the nil values.

               
```
 var title: String?
var creator: String?
```